packages/lesswrong/components/forumEvents/ForumEventPoll.tsx

```diff
@@ -652,72 +691,33 @@ export const ForumEventPoll = ({
           <DeferRender ssr={false}>
             {!hideViewResults && !resultsVisible && (
               <div className={classes.votePrompt}>
-                {voteCount > 0 &&
-                  `${voteCount} vote${voteCount === 1 ? "" : "s"} so far. `}
-                {hasVoted
-                  ? "Click and drag your avatar to change your vote, or "
-                  : "Place your vote or "}
-                <button
-                  className={classes.viewResultsButton}
-                  onClick={() => setResultsVisible(true)}
-                >
-                  view results
+                {voteCount > 0 && `${voteCount} vote${voteCount === 1 ? "" : "s"} so far. `}
+                {hasVoted ? "Click and drag your avatar to change your vote, or " : "Place your vote or "}
+                <button className={classes.viewResultsButton} onClick={() => setResultsVisible(true)}>
+                  view results.
```

Comment:
This tooltip made it look too busy when the other two icons are present (close button and toggle comment form button). These only appear after you have made one vote, so I've removed this tooltip in that case (I hope the click-and-drag behaviour will be self explanatory)